lib/commands/web.js

```diff
@@ -66,4 +66,24 @@ extensions.checkForAlert = async function () {
   return false;
 };
 
+extensions.waitForAtom = async function (promise) {
```

Comment:
@jlipps : I think you are right. I just copied as is from `Ios-driver`. We can get rid of rest of stuff as you said, only thing we wont have `Error received while executing atom` in error message